sdk/core/Azure.Core/tests/FixedRetryPolicyTests.cs

```diff
@@ -54,12 +55,34 @@ public async Task WaitsSameAmountEveryTime()
             Assert.AreEqual(500, response.Status);
         }
 
+        [Test]
+        public async Task WaitsSameAmountBetweenTries_Exceptions()
```

Comment:
This test exercises a fix in this PR. Previously, the delay would always be exponential (even when the mode was Fixed) when computing the delay after an exception - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/Pipeline/Internal/RetryPolicy.cs#L204.